scipy/stats/stats.py

```diff
@@ -346,6 +346,26 @@ def _broadcast_shapes_with_dropped_axis(a, b, axis):
     return shp
 
 
+def _convert_symmetric_p_value(pvalue, root_func, alternative):
+    """
+    Convert a one-sided, symmetric P-value to a one or two-sided p-value. root_func
+    describes the source distribution function ('sf' or 'cdf')
+    """
+    if alternative not in ["two-sided", "greater", "less"]:
```

Comment:
This is a private function, so only developers will use it directly, and so you don't have to do input validation.

If you were to do input validation:
```suggestion
    if alternative not in {"two-sided", "greater", "less"}:
```
Lookup is O(1) on average because they're implemented using a hash table; lookup in a list is O(n) because they're basically fancy arrays. 
Of course it doesn't matter in practice, just a matter of principle and style.